README.md

```diff
@@ -1,5 +1,21 @@
-# movemeet
+# *movemeet*
 
-### SDP spring semester 2021
+### SDP - spring semester 2021
 
 [![Build Status](https://api.cirrus-ci.com/github/movemeet/movemeet.svg)](https://cirrus-ci.com/github/movemeet/movemeet)
+
+This app was created for the CS-306 Software Development Project course.
+
+*movemeet* is an Android application allowing you to organize any type of sports activity. The user can either join acitivities or plan an activity at the location he wants directly via an interactive map. He can then add a description to his activity and invite partners to join him! Each activity has its own chat allowing participants to communicate in a simple and efficient way. The app includes additional features such as workout exercises and the possibility to record your GPS track during a run or a walk.
```

Comment:
It's a good idea to have a readme explaining the purpose of the app, as the code alone is not always enough to fully understand what we want to achieve :)
PS: careful, in the second sentence, you wrote "acitivities" instead of "activities".

Reply:
Nice comment! Thank you for re-reading the README! I will modify this directly!